adding_metadata_betydb_yaba.md

```diff
@@ -0,0 +1,27 @@
+## BETYdb-YABA and Python Client Library
+
+BETYdb-YABA and its python client library provides an automated bulk upload to many BETYdb tables. API endpoints have been implemented to upload data to respective tables. For more information including examples for all endpoints and a link to the Swagger documentation, see the BETYdb-YABA [README](https://github.com/PecanProject/BETYdb-YABA/blob/master/README.md){target="_blank"}.
+
+Here are some examples of how to hit Experiments and Sites endpoints:
+
+* Experiments:
+
+```sh
```

Comment:
If you indent each of these code blocks four spaces, they will appear nested under their respective items.